config/locales/en.yml

```diff
@@ -124,8 +124,27 @@ en:
   mediated_pages:
     header:
       default: 'Request on-site access'
-      HOPKINS: 'Request delivery to campus library'
-      PAGE-MP: 'Request delivery to campus library'
+      HOPKINS: 'Request delivery to campus library' # Deprecated
+      PAGE-MP: 'Request delivery to campus library' # Deprecated
+      EAR-PAGE-MP: 'Request delivery to campus library'
+      SAL3-PAGE-MP: 'Request delivery to campus library'
+      MAR-ATLASES: 'Request delivery to campus library'
```

Comment:
I hope we can avoid playing new location whack-a-mole if/when Hopkins adds more locations; is there any way we can continue to use the library code?

Reply:
Okay, I reverted this.  I thought this way was less complex. 🤷